sycl/ReleaseNotes.md

```diff
@@ -91,7 +91,7 @@ enviornment variable. [bd03e0d3]
  extension specification. [7f2b17ed]
 - Added the [`sycl_ext_oneapi_memcpy2d`](doc/extensions/supported/sycl_ext_oneapi_memcpy2d.asciidoc)
 extension specification. [296e9c3a]
-- Added the [`sycl_ext_oneapi_user_defined_reductions`](doc/extensions/experimental/sycl_ext_oneapi_group_sort.asciidoc)
+- Added the [`sycl_ext_oneapi_user_defined_reductions`](doc/extensions/proposed/sycl_ext_oneapi_user_defined_reductions.asciidoc)
```

Comment:
Not sure if it should be fixed, because commit merged yesterday, but now this doc in doc/extensions/experimental

https://github.com/intel/llvm/pull/7850

Reply:
The links are bound to get outdated at some point, but since I'm touching these anyways, makes sense to me to update them.